tests/scripts/configure-rancher-step-01

```diff
@@ -6,7 +6,8 @@ set -e -x
 REG_NAME=machine-registration
 
 # Install generated RancherOS Operator chart
-helm upgrade --install rancheros-operator ../artifacts/rancheros-operator-*.tgz \
+helm repo add rancheros https://rancher-sandbox.github.io/rancheros-operator/
+helm upgrade --install rancheros-operator rancheros/rancheros-operator --devel \
```

Comment:
nit: ditto here, but here it's for testing.. as we loose the tgz at hand, maybe we should or require a chart url, or a version